code/stable/glassbr/SRS/GlassBR_SRS.tex

```diff
@@ -492,13 +492,13 @@ \subsubsection{Data Definitions}
               \item{$J$ is the stress distribution factor (Function) (Unitless)}
               \end{symbDescription}
 \\ \midrule \\
-Notes & $a$ and $b$ are dimensions of the plate, where ($a\geq{}b$).
+Notes & the $a$ and $b$ are dimensions of the plate, where ($a\geq{}b$).
```

Comment:
I think we should remove the "the" here; it sounds awkward.

Reply:
Good catch! It's supposed to be "$a$ and $b$ are the dimensions of the plate, where ($a\geq{}b$)."

Reply:
Fixed in 5b307d947.

Reply:
👍